usage.go

```diff
@@ -98,8 +98,59 @@ func formatType(buf *bytes.Buffer, unformattedType []byte) {
 	_, _ = buf.Write(formattedType[len(typePrefix):])
 }
 
+func dumpMethods(buf *bytes.Buffer, typ reflect.Type) {
```

Comment:
@markdryan when would one use this. I assume this is just a way to dump a type completely.